### DISCUSSION TITLE
Contributing using Windows

### DISCUSSION THREAD

**Discussion by g20yr** [adidas/lakehouse-engine#11] [Q&A]
Hello Everyone,

I want to contribute to this repo and this is my first time working on something for open source. I cloned the repo in my visual studio but some how unable to start the virtual environment. Please suggest if i am missing anything in here.

**Comment by PMRocha** [comment]
Hello @g20yr, 

first off welcome aboard.
Regarding your issue it would be helpful to share what kind of error you are having.
On windows we have a different make command to create the docker image (build-image-windows and not build-image).
When it comes to venv it might be caused to the requirements file being inside the cicd folder and not in the root of the repo (just spit balling here).

**Comment by g20yr** [comment]
Hello @PMRocha, 

Could you please share any docs related to setting up the requirements in Windows if you have any or please share me any blogs that detail this process.
